packages/features/embed/Embed.tsx

```diff
@@ -545,6 +545,9 @@ const EmbedTypeCodeAndPreviewDialogContent = ({
     { enabled: !!eventId && embedType === "email", refetchOnWindowFocus: false }
   );
 
+  const { data: user } = trpc.viewer.me.useQuery();
```

Comment:
@hariombalhara do we want to send the whole `/me` endpoint to the embed component?

Reply:
Thanks for highlighting this @joeauyeung. First of all, we shouldn't do it here. There are wrapper components that provide all the data to Embed, EventTypeEmbed.tsx and RoutingFormEmbed.tsx. They should be responsible to do that and pass a `defaultBrandColor` prop

I think it's okay to do reuse viewer.me query as it's anyways fetched at a lot of places.